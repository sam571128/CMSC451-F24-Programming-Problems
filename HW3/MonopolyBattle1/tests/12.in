53 2809 1000
12 9
15 50
37 5
14 29
43 19
26 39
36 30
18 35
38 12
30 34
13 40
27 9
47 4
27 35
51 51
41 26
29 23
48 32
39 36
13 12
13 32
14 22
32 45
43 24
32 31
1 14
14 2
18 1
8 50
50 38
49 6
7 3
29 24
34 25
40 23
36 26
43 14
37 48
14 19
24 27
30 41
29 30
41 47
28 47
9 15
38 15
15 51
33 13
30 10
33 47
39 14
26 44
32 13
47 18
26 19
43 30
49 44
43 15
14 5
31 9
43 25
28 4
36 12
2 2
52 33
49 53
52 32
25 13
24 38
26 25
32 43
16 49
42 15
29 47
13 45
42 12
35 5
33 11
13 48
40 35
6 16
36 34
25 9
43 3
16 9
11 2
37 39
38 8
40 1
51 28
29 50
8 11
53 12
5 40
47 8
2 11
12 13
52 52
32 34
27 1
4 50
33 39
14 42
9 27
49 35
25 46
29 31
50 10
40 29
30 27
48 7
50 48
36 41
46 29
53 4
13 22
51 35
4 34
48 27
52 53
50 30
34 53
5 45
21 31
1 8
6 4
8 51
39 27
20 33
40 17
38 45
7 24
51 23
6 27
6 42
14 51
35 26
5 49
35 45
15 29
51 33
23 43
2 3
19 22
4 48
16 23
31 42
18 38
38 2
48 50
35 20
19 46
30 40
10 35
5 22
45 21
46 44
3 20
8 39
41 33
4 41
15 14
11 23
43 44
48 9
46 2
34 37
28 3
42 47
10 14
40 42
27 11
45 5
29 34
16 29
52 5
40 53
49 41
31 14
46 4
8 9
33 51
47 51
28 43
34 28
14 35
36 31
44 38
37 10
10 41
28 30
49 27
25 23
25 18
22 40
33 12
19 9
46 26
28 21
51 44
3 34
41 27
43 28
52 24
45 28
32 28
40 2
13 36
51 41
23 23
51 3
13 52
22 15
13 6
7 37
44 32
18 43
27 14
20 32
50 40
5 27
43 22
52 30
23 1
41 16
16 26
9 32
52 28
3 30
4 44
13 39
30 16
22 41
41 14
36 38
31 46
36 32
22 12
20 36
20 39
53 8
3 15
18 46
51 43
49 31
43 27
8 20
12 24
22 16
26 51
20 19
1 22
6 34
44 9
48 1
14 32
28 13
46 5
47 52
24 36
2 23
21 10
33 46
46 14
26 9
19 2
11 35
53 45
48 12
45 49
45 48
5 26
10 5
19 18
27 32
35 52
33 31
14 37
14 33
50 5
24 19
20 26
29 26
39 28
22 2
22 6
11 24
47 44
5 31
40 20
29 33
5 9
53 14
12 31
12 1
38 35
42 2
27 44
24 41
34 40
52 27
49 8
45 39
41 12
53 38
31 7
21 9
14 30
41 22
27 24
7 47
7 34
38 20
12 47
24 44
3 38
15 38
23 35
50 11
45 47
4 49
7 15
5 46
31 39
21 38
16 34
46 20
19 36
46 21
4 3
10 49
13 42
3 33
40 14
8 24
4 47
49 23
23 6
32 33
35 1
32 42
21 14
30 13
45 11
12 7
30 35
52 40
37 27
24 2
31 8
40 22
32 52
7 21
27 4
52 15
44 48
4 53
5 5
24 23
21 24
31 45
17 35
30 45
26 32
24 10
21 3
1 38
31 53
6 33
47 21
35 46
40 47
42 25
31 19
30 26
26 53
39 22
17 28
6 3
22 19
49 22
19 8
29 3
30 2
28 35
27 48
32 25
48 24
23 21
2 4
36 18
29 49
36 1
39 3
27 42
16 43
38 47
51 11
34 9
20 37
19 31
3 14
31 24
28 18
37 36
26 3
25 4
27 18
18 7
9 48
13 25
35 23
26 14
45 26
30 37
18 40
47 11
10 34
50 31
37 8
45 24
4 35
11 27
49 45
49 4
1 46
8 16
50 35
1 21
26 34
17 15
8 23
13 7
17 21
24 32
10 20
34 36
45 22
22 9
32 6
49 26
50 53
1 13
10 37
9 47
1 39
41 19
26 42
35 2
29 8
22 17
30 51
1 51
19 51
20 52
44 14
52 38
11 50
51 12
6 17
44 31
34 10
46 25
50 2
10 52
10 9
50 50
9 22
42 19
14 31
43 21
24 43
23 14
5 8
48 45
39 51
34 19
16 1
37 40
35 15
29 48
37 1
26 37
6 46
32 10
6 47
43 34
13 21
50 19
43 48
12 12
5 6
20 6
34 48
26 43
10 26
13 24
52 48
44 3
42 39
13 5
33 35
42 48
52 51
28 32
38 19
4 43
51 46
11 14
49 20
23 37
7 22
30 17
10 4
32 2
53 29
4 42
20 42
22 18
27 5
33 22
7 49
33 24
26 18
51 19
26 40
31 41
13 8
39 1
8 46
13 53
1 43
22 51
7 32
43 1
50 1
46 51
44 5
26 41
43 18
25 37
41 35
34 16
1 28
9 42
15 30
47 9
13 49
48 35
11 34
30 46
1 42
25 34
16 41
30 19
53 6
30 33
28 17
5 25
51 53
17 7
43 49
36 46
25 53
35 33
45 44
8 44
18 33
33 2
34 46
23 5
43 4
12 37
45 32
17 13
22 21
7 17
44 13
25 36
49 28
2 15
41 25
19 45
44 49
46 41
27 19
37 3
8 18
15 7
38 40
26 36
6 9
47 37
30 1
42 8
41 2
29 7
27 22
4 1
33 6
13 43
1 44
26 47
42 18
15 42
19 37
28 34
7 45
7 25
2 10
27 28
6 41
7 41
33 28
53 35
10 29
27 53
34 5
7 42
53 37
28 38
15 32
17 40
30 39
9 18
28 12
26 10
34 51
11 47
16 30
52 23
31 3
24 39
25 11
38 43
9 41
41 17
22 26
41 20
40 7
11 25
10 22
50 23
38 18
27 50
37 33
34 21
17 34
2 48
46 32
39 9
22 23
31 44
2 45
18 3
35 42
14 12
42 36
28 1
47 28
39 39
12 48
20 28
2 1
36 4
33 48
46 10
4 7
30 53
51 9
14 43
40 50
20 38
11 15
46 11
51 34
15 44
2 52
3 12
42 1
37 42
19 11
42 41
36 14
46 30
47 35
51 1
25 15
38 38
4 37
25 50
12 29
43 51
31 12
44 30
21 4
48 53
20 21
28 5
49 5
21 2
7 12
6 51
21 33
15 45
11 8
23 48
18 48
18 32
4 13
25 38
11 33
39 2
1 24
10 13
5 32
36 45
10 36
19 44
15 27
37 43
20 51
23 42
5 2
34 23
8 28
38 33
36 24
18 16
27 21
30 48
50 20
25 10
50 28
41 18
21 52
44 23
53 48
44 16
39 46
35 19
35 21
38 49
32 11
37 37
43 26
11 48
51 24
33 25
19 19
32 7
28 25
31 26
24 42
48 2
19 35
47 5
28 15
2 43
16 27
18 15
6 11
18 47
11 4
1 10
1 27
16 8
20 2
19 40
42 46
40 15
19 34
8 2
37 32
43 10
6 19
42 24
25 33
52 22
19 14
18 28
24 50
3 43
26 52
48 33
19 28
16 42
25 47
8 38
14 36
22 11
12 39
3 46
38 52
33 36
35 4
10 15
21 53
21 20
27 51
30 38
24 18
46 22
42 44
53 51
38 13
22 43
13 41
52 47
34 43
21 36
25 8
46 33
13 37
32 32
50 34
16 22
34 24
42 17
48 49
37 16
13 26
17 23
8 17
16 32
5 48
49 12
38 32
3 48
26 35
15 41
30 11
5 24
18 11
31 20
32 53
49 13
45 30
19 41
22 46
43 8
23 28
35 41
39 38
8 12
9 2
39 16
39 37
19 23
23 12
52 41
9 26
20 35
29 52
46 9
28 24
39 21
18 22
11 49
24 9
4 30
52 49
49 15
46 24
36 43
10 25
2 20
19 50
53 27
37 38
53 42
25 5
2 12
18 24
52 43
17 1
9 14
9 7
49 40
49 39
5 20
50 3
2 38
48 6
50 7
4 15
24 11
44 46
5 51
47 19
28 52
33 18
17 20
31 18
7 31
20 29
50 51
38 53
15 34
19 7
52 11
8 47
24 33
41 13
26 46
29 25
3 11
21 32
9 29
36 17
23 32
9 30
16 37
45 17
1 12
32 49
40 38
34 49
53 32
52 3
35 28
45 19
27 7
29 51
30 32
37 9
12 27
18 45
17 25
12 26
28 11
24 5
12 30
41 10
38 3
17 43
20 40
50 24
43 33
5 37
42 32
25 21
6 45
2 17
9 6
53 52
12 28
15 36
47 29
35 35
34 50
48 16
31 32
21 50
18 36
46 46
5 35
31 43
43 7
47 12
21 17
9 33
11 43
36 19
23 13
20 53
3 10
32 38
53 20
47 23
44 28
50 41
47 26
46 45
44 25
49 51
23 38
35 34
15 8
40 12
29 39
36 48
29 1
11 1
4 38
35 17
42 31
46 35
19 10
49 32
5 53
28 44
41 34
38 25
28 51
50 13
18 29
47 17
1 18
5 33
33 26
31 11
45 10
50 44
15 37
38 14
43 38
24 24
5 42
23 4
18 8
6 43
27 2
46 53
19 25
34 35
16 48
40 18
15 15
30 5
7 51
53 31
50 33
31 35
35 37
53 11
27 25
24 15
48 46
38 31
44 33
9 46
43 41
17 6
31 13
14 15
45 16
10 48
13 23
41 40
3 39
17 19
3 31
5 43
36 47
8 35
13 13
43 52
23 25
21 5
4 10
12 38
4 27
25 7
38 21
44 4
52 19
51 45
45 34
18 27
18 52
42 3
2 13
30 23
38 48
16 7
20 12
27 30
30 36
3 26
52 12
10 33
31 47
28 16
23 52
9 10
45 51
36 42
2 35
4 46
53 3
10 32
10 27
34 52
20 46
32 17
6 48
38 30
29 18
26 22
48 34
48 31
14 25
52 10
45 1
21 11
2 42
12 16
39 34
12 53
53 34
28 42
7 35
7 52
3 53
51 5
32 20
53 15
35 7
25 28
11 6
9 16
53 26
3 44
51 47
9 1
37 24
9 49
18 18
7 33
18 25
13 34
35 31
46 27
45 52
26 45
42 23
40 31
13 51
48 47
29 27
42 50
42 9
11 32
53 50
24 40
45 18
23 24
41 51
49 29
40 36
9 24
41 7
1 48
51 15
51 10
1 25
31 30
22 35
33 3
2 49
31 34
35 14
18 21
3 35
42 5
28 10
28 29
41 37
3 32
3 37
20 10
38 16
36 27
15 24
31 25
9 34
45 7
44 7
40 16
4 39
32 3
33 32
42 51
12 46
52 36
43 17
33 9
8 40
32 44
52 29
48 18
21 6
45 50
51 40
37 14
14 20
10 16
2 31
36 13
9 53
28 45
22 22
11 22
28 19
32 29
47 32
53 28
4 11
45 38
1 35
34 17
12 42
11 21
15 39
23 3
17 32
41 11
35 40
31 51
21 47
22 14
39 47
12 40
14 8
28 50
13 35
4 16
20 48
1 40
33 17
27 15
53 10
29 15
41 50
11 53
32 16
32 1
23 39
37 31
19 6
4 40
27 45
40 19
36 50
4 8
33 34
20 34
28 8
49 19
44 36
52 20
12 25
31 6
49 33
19 15
23 17
45 12
34 44
5 41
44 8
28 14
22 42
20 17
2 44
18 26
27 47
3 29
6 12
8 4
53 16
14 39
3 42
39 4
33 20
2 33
14 52
6 38
45 15
2 30
15 49
44 12
6 24
17 31
25 42
19 24
15 18
19 16
15 12
4 29
11 26
41 31
34 33
36 40
48 52
32 15
10 40
32 18
53 22
18 19
46 37
22 37
53 2
16 15
47 7
31 40
36 37
6 20
36 39
16 6
51 27
16 36
33 33
49 49
34 6
32 8
1 3
29 44
36 2
52 34
20 1
50 43
25 44
2 39
40 52
50 8
5 39
45 27
13 17
53 24
24 46
27 34
44 50
46 48
1 26
10 7
30 3
27 33
26 6
27 39
6 50
35 18
4 52
30 50
8 36
13 9
7 4
26 48
15 35
24 3
14 10
14 11
21 28
44 20
8 48
33 43
45 43
9 11
33 7
36 7
31 17
37 21
6 10
9 13
26 8
9 23
51 20
53 9
2 41
21 23
48 23
38 36
11 7
8 26
3 40
12 35
46 1
15 31
35 44
52 9
4 33
5 4
21 12
5 19
52 44
12 52
2 34
18 23
6 39
53 46
46 38
31 21
47 33
46 12
45 8
28 33
18 12
28 41
13 50
11 5
17 29
28 48
25 25
14 46
52 37
10 18
44 19
17 52
18 53
31 4
13 44
26 17
15 17
25 51
44 6
15 1
34 1
5 44
52 8
40 4
19 12
38 10
12 3
21 51
27 17
38 5
14 49
41 41
9 38
42 10
41 28
3 45
27 46
4 21
36 51
10 44
24 1
7 30
42 52
2 47
20 43
3 16
38 24
25 52
2 28
21 26
28 23
45 3
31 22
24 48
24 14
33 37
28 26
34 7
2 29
10 43
15 9
40 30
47 27
34 15
1 23
35 38
18 13
2 6
26 29
12 36
10 53
34 38
53 7
41 8
42 27
21 21
17 26
10 21
43 13
53 30
38 9
19 26
44 17
9 43
22 52
14 13
52 45
18 20
12 5
47 49
32 48
52 42
29 38
3 13
41 49
19 33
43 46
45 9
22 39
31 16
22 24
10 17
38 11
7 19
33 16
37 41
52 31
37 46
23 45
42 22
39 43
47 50
45 41
26 30
18 34
11 52
39 24
43 42
22 5
26 4
42 53
53 40
21 41
3 8
27 38
17 2
52 18
47 20
21 46
40 24
35 13
14 18
33 44
43 53
8 29
41 5
6 40
46 39
33 8
13 2
29 5
21 25
9 31
24 47
12 22
16 4
30 31
17 53
4 4
3 6
45 37
8 53
30 6
43 12
11 38
4 18
23 33
32 9
25 39
5 3
11 36
11 39
43 29
12 32
14 44
28 31
21 29
8 31
21 35
42 45
37 2
22 34
38 46
25 17
53 23
17 16
9 44
4 45
8 7
48 10
47 3
7 16
1 20
33 45
22 7
41 3
50 52
44 51
5 36
21 45
1 45
17 12
23 26
51 8
40 10
31 31
28 36
39 53
44 53
24 4
34 2
41 39
44 39
38 28
28 7
21 7
14 48
45 53
14 27
35 24
49 48
1 47
5 23
12 6
43 11
5 11
40 5
42 34
25 35
13 14
14 45
38 22
1 17
2 8
34 8
9 12
28 9
22 3
18 4
43 36
10 51
46 8
32 47
37 30
28 39
34 13
2 37
40 11
39 45
52 25
45 6
21 43
29 10
52 21
7 13
2 53
15 26
8 10
23 20
15 11
45 2
30 12
24 35
38 23
49 21
7 39
51 13
30 8
17 41
7 29
44 18
50 29
30 9
5 30
34 42
22 1
8 14
5 34
10 8
30 14
37 34
41 6
11 11
16 31
12 18
13 20
40 3
51 26
7 11
23 49
1 33
7 28
34 3
35 43
50 17
37 18
42 16
21 39
36 25
36 35
20 50
42 4
52 46
17 11
50 45
2 26
2 36
19 29
53 19
49 3
20 20
50 32
1 1
33 49
45 14
16 21
13 3
49 14
29 29
32 24
51 42
14 53
20 30
39 6
33 41
6 23
21 22
18 14
4 23
42 35
53 41
23 30
2 24
5 29
40 39
40 40
49 1
37 44
5 13
7 14
16 40
25 32
5 38
47 46
21 42
46 42
49 24
16 53
22 36
8 52
30 49
2 19
37 23
19 38
3 2
7 53
50 39
23 53
12 10
25 6
48 30
11 44
19 49
47 34
2 14
17 42
23 10
34 11
47 41
38 26
51 16
19 42
49 11
48 8
3 28
13 31
52 2
44 42
7 9
39 42
18 41
20 13
40 25
7 27
44 37
3 51
22 47
37 50
13 33
15 23
5 14
19 47
25 24
27 27
36 6
19 39
50 25
44 11
4 20
43 37
12 4
4 19
43 9
40 32
28 22
47 22
13 16
20 24
29 32
51 32
30 44
24 49
13 4
18 39
23 22
13 19
45 46
23 34
6 37
36 10
3 25
43 39
15 5
20 7
34 14
6 32
51 30
17 14
11 10
14 6
44 40
26 26
25 22
46 18
17 4
9 52
36 52
35 22
14 50
9 36
8 22
8 21
49 7
51 17
50 21
33 15
35 9
27 29
39 15
13 15
19 53
35 11
50 6
41 44
44 24
32 37
46 50
20 47
9 9
2 25
17 22
29 43
29 22
30 24
38 51
51 48
41 15
18 50
40 44
44 44
53 53
22 48
39 33
22 10
3 22
39 8
16 35
51 6
23 8
10 50
52 35
26 7
38 17
7 26
47 40
42 20
30 20
15 13
37 45
45 13
9 21
15 3
44 35
33 21
43 31
29 2
5 15
24 17
47 14
1 7
26 23
21 44
23 41
53 21
22 50
5 1
40 37
23 2
1 50
32 23
23 47
14 26
34 12
4 17
19 4
49 37
41 24
12 45
26 12
32 50
34 31
39 25
53 18
32 4
15 21
4 6
39 17
11 41
24 34
41 21
32 46
18 30
12 11
23 15
40 6
51 4
21 27
49 30
40 27
24 20
6 44
40 43
20 15
2 51
12 33
22 44
16 28
52 14
1 9
40 28
36 15
20 11
22 33
21 30
17 17
25 3
39 35
29 9
18 37
8 25
42 13
46 6
8 34
39 41
39 40
10 42
47 6
19 13
40 13
14 16
21 16
22 8
42 38
29 16
37 13
3 5
28 49
46 40
23 18
19 32
31 5
21 48
2 16
17 24
49 47
9 50
47 16
6 7
47 48
12 17
46 34
47 10
16 44
22 30
24 7
50 36
4 28
34 39
47 42
21 37
51 29
49 2
40 34
26 20
20 45
39 13
16 52
24 53
6 29
29 4
9 51
11 30
18 5
52 1
35 32
50 4
32 14
28 20
23 40
7 23
8 3
3 4
6 1
36 16
32 35
12 51
10 28
41 53
33 1
3 7
3 24
48 37
3 52
18 10
9 8
36 44
14 28
35 25
27 36
26 1
26 16
18 2
22 4
12 14
17 45
24 45
7 7
16 18
8 19
32 26
4 12
14 4
52 39
18 9
33 30
16 17
12 34
3 23
26 31
29 13
48 43
44 29
6 8
16 5
26 21
52 6
21 15
46 15
11 37
24 37
15 52
31 10
16 10
41 1
24 26
39 30
43 20
53 39
18 51
16 16
37 6
31 1
18 6
34 22
44 41
1 6
30 47
15 4
1 4
41 38
10 23
29 36
15 53
33 38
22 25
24 12
24 25
47 43
42 33
10 38
1 37
19 27
45 42
17 30
32 40
33 10
25 1
29 17
22 31
31 37
37 11
33 42
7 2
30 22
6 49
53 33
18 17
53 49
23 50
26 5
8 13
40 45
10 19
3 21
44 22
51 14
48 44
32 12
25 20
46 28
4 22
17 49
32 21
13 11
19 30
43 45
29 53
11 40
38 6
53 13
34 32
45 4
31 50
14 21
47 13
38 44
9 5
7 46
17 36
30 7
12 49
16 25
27 37
50 12
43 6
37 19
25 29
35 36
28 2
8 5
16 3
47 25
49 38
37 29
30 25
8 41
42 43
29 14
15 43
5 18
20 27
31 2
51 2
41 42
25 31
27 43
51 37
21 19
6 18
53 36
27 8
8 42
39 49
47 30
40 51
36 3
37 49
1 36
32 19
5 12
37 53
10 12
35 10
43 5
48 17
36 20
40 48
20 8
50 14
42 37
3 49
26 49
26 15
23 11
27 49
48 26
10 3
36 49
2 9
27 23
22 32
20 49
40 21
5 28
28 37
15 2
37 47
6 6
1 31
44 10
13 18
32 30
6 52
5 21
6 5
50 15
48 29
25 12
22 20
43 32
35 16
12 44
49 16
34 27
41 45
36 8
35 6
46 13
39 20
7 38
40 26
45 23
37 22
46 36
13 27
37 25
36 28
10 2
30 4
5 47
52 50
47 38
6 25
15 22
17 44
50 47
38 7
33 27
28 46
13 29
44 52
1 16
23 51
17 47
19 21
42 7
46 23
44 27
39 32
16 24
27 20
14 9
34 26
25 19
53 25
38 37
51 39
31 49
51 7
7 8
25 41
35 30
21 49
18 49
3 27
48 20
15 20
49 36
1 29
48 42
30 28
3 9
9 4
35 3
11 28
37 35
20 3
47 45
11 12
24 8
25 49
42 49
7 36
48 22
40 33
41 36
42 26
48 11
8 32
16 11
14 1
49 42
1 11
27 31
48 41
53 17
29 6
28 27
32 51
38 27
7 20
11 3
32 39
22 13
3 36
1 49
24 16
31 29
6 35
51 52
39 7
13 47
3 1
49 46
39 19
49 43
6 22
10 46
1 19
17 9
48 5
34 29
29 37
16 19
13 38
39 10
8 45
19 5
29 11
49 18
4 25
4 51
40 41
14 14
6 53
49 17
48 38
48 21
38 29
47 31
10 10
49 50
46 47
23 44
13 46
52 16
15 40
34 20
30 15
4 5
45 33
23 19
53 43
14 23
42 40
41 32
50 18
30 29
9 28
38 41
42 11
13 30
14 3
27 12
52 17
8 27
8 43
24 22
48 28
16 20
39 18
25 45
23 7
38 34
37 20
26 33
48 25
30 43
41 52
46 7
19 17
51 38
19 52
37 17
29 12
17 33
36 36
37 52
27 13
36 33
48 48
17 8
44 47
4 24
2 50
42 14
33 5
26 2
11 31
18 42
22 49
14 24
10 31
46 43
41 43
16 12
24 13
6 26
35 47
15 47
1 2
4 14
32 27
6 21
12 43
15 46
47 15
26 28
39 44
29 41
16 38
35 53
47 53
40 9
44 2
12 21
27 10
14 41
35 8
29 40
35 39
26 50
23 29
24 28
34 30
44 43
38 4
25 16
2 21
45 20
35 50
51 31
22 28
17 46
20 41
25 30
2 32
20 9
50 26
39 5
47 24
25 48
48 51
21 34
22 53
46 49
16 47
27 41
13 28
48 15
49 34
15 16
11 29
8 15
48 40
26 13
10 39
39 26
33 40
16 39
40 49
20 18
19 48
46 3
43 43
30 30
33 29
6 31
7 18
27 16
46 17
4 31
16 2
45 45
4 36
5 52
24 30
4 32
25 2
3 17
29 28
35 51
53 5
34 34
20 23
19 3
23 36
24 51
32 5
50 27
28 6
17 37
51 21
8 49
9 40
44 45
29 42
51 25
33 23
17 50
50 22
11 19
29 45
11 51
41 48
35 27
51 49
12 20
41 46
47 39
20 5
11 18
4 26
52 7
33 4
7 6
37 51
15 19
47 2
50 9
11 9
20 4
41 29
16 14
21 8
22 45
36 5
17 38
14 38
30 21
37 4
7 48
48 36
16 50
3 18
26 11
5 17
31 38
44 21
45 31
21 13
28 53
41 4
14 34
11 17
30 52
17 18
27 26
25 40
1 15
29 35
9 45
27 6
30 42
10 30
46 31
37 26
21 1
24 21
15 10
34 18
16 45
45 25
23 9
51 36
45 36
10 45
6 28
47 1
1 30
19 43
27 40
38 50
12 23
16 13
9 17
42 21
39 48
31 27
24 6
34 45
8 37
26 24
2 46
38 1
18 44
12 19
49 52
13 10
47 36
6 30
37 15
51 18
24 52
1 34
8 1
33 50
34 47
12 15
30 18
48 39
26 27
22 27
41 23
42 28
46 16
7 44
9 3
44 1
17 27
5 16
12 50
36 11
24 31
6 14
14 47
44 34
11 42
21 40
22 29
39 23
53 1
11 20
14 7
51 50
23 46
31 33
2 22
43 16
20 44
48 19
49 9
8 33
48 3
38 39
20 25
43 2
35 29
17 51
3 3
51 22
28 40
23 31
41 30
10 6
17 3
14 17
52 13
48 4
36 22
44 26
40 8
52 26
29 21
39 31
43 40
12 8
7 1
20 22
37 12
15 6
19 1
48 14
34 4
12 41
29 19
42 29
45 29
7 40
31 48
43 23
11 45
9 37
53 47
15 28
6 13
7 50
49 25
16 51
25 27
9 20
7 5
42 30
27 52
32 36
7 10
1 52
21 18
33 52
3 50
25 43
46 19
35 48
39 29
15 48
2 27
43 50
11 16
42 42
31 28
3 19
50 46
25 26
11 46
23 27
46 52
36 9
23 16
18 31
39 12
5 50
37 28
32 41
32 22
31 23
29 20
40 46
50 37
3 47
9 35
20 31
37 7
14 40
45 35
10 47
17 39
39 11
36 29
20 16
1 53
16 33
24 29
1 41
6 2
13 1
10 24
1 32
10 11
17 48
4 2
49 10
12 2
6 36
22 38
2 7
26 38
39 50
48 13
50 49
31 36
17 10
31 52
6 15
43 47
8 30
9 39
50 16
19 20
10 1
2 40
47 47
29 46
17 5
39 52
27 3
38 42
33 19
5 10
15 25
16 46
1 5
3 41
20 14
2 18
36 21
34 41
44 15
2 5
9 19
8 6
36 53
43 35
4 9
5 7
15 33
25 14
7 43
45 40
9 25
52 4
33 53
53 44
35 49
8 8
31 15
33 14
11 13
35 12
41 9
42 6
36 23
50 42
28 28
