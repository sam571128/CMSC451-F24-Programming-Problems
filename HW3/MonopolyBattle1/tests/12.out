54 51 84 43 59 138 111 88 45 49 92 46 68 51 46 138 130 54 51 151 156 51 51 51 42 42 51 90 51 42 68 68 141 42 51 42 167 46 42 68 42 103 71 52 68 106 43 79 138 46 47 212 122 